File: rb/utility/make_context.rb
# MagicTheGatheringTwo SDK utility: make_context
require_relative '../core/context'
module MagicTheGatheringTwoUtilities
  MakeContext = ->(ctxmap, basectx) {
    MagicTheGatheringTwoContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// MagicTheGatheringTwo SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class MagicTheGatheringTwoMakeContext
{
    public static function call(array $ctxmap, ?MagicTheGatheringTwoContext $basectx): MagicTheGatheringTwoContext
    {
        return new MagicTheGatheringTwoContext($ctxmap, $basectx);
    }
}
